CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+This is a simple guide of the workflow being used in Cryostat. We will use https://github.com/cryostatio/cryostatio.github.io as an example.
+
+## Fork the upstream repository
+
+See https://docs.github.com/en/get-started/quickstart/fork-a-repo
+
+## Local changes
+
+Clone the forked repository.
+```bash
+$ git clone git@github.com:<username>/cryostatio.github.io.git
+$ git remote add upstream git@github.com:cryostatio/cryostatio.github.io.git # Adding upstream as remote
+```
+
+Checkout another branch.
+```bash
+$ cd cryostatio.github.io && git checkout -b some-task-branch
+```
+
+Edit some files and commits your changes.
+```bash
+$ git add . # Stage your changes. Here, we stage the entire repo (recursively).
+$ git commit -m "some very nice message"
+```
+
+## Open a PR
+
+Push your changes to your remote respository (forked). You might need to rebase on the newest commit on the upstream.
+
+To rebase:
+```bash
+$ git fetch upstream # Fetch upstream commits
+$ git checkout main && git rebase upstream/main && git push origin main # Checkout main, merge upstream and update origin
```

Comment:
I think you can just 
```
To rebase: 
$ git fetch upstream
$ git rebase upstream/main
```
```
To push:
$ git push -f
```

I see that this you are rebasing your own fork first, so maybe mention that's what is happening? Like
```
To rebase your fork with the latest changes: 
...
...
# Checkout main, rebase on top of upstream, pull from origin
```
Or something like that.

Reply:
Make sense to me! I will fix this.